app/(main)/layout.tsx

```diff
@@ -30,12 +30,19 @@ export default function MainPageLayout({
     >
       <Background />
       <Header>
-        <Menu />
-        <SocialLinks />
```

Comment:
HeaderNav components contain items and are positioned via passed classNames

Reply:
Probably best to change these labels to constants, but there's a separate ticket for that.